src/mlpack/tests/rbm_network_test.cpp

```diff
@@ -0,0 +1,246 @@
+/**
+ * @file rbm_network_test.cpp
+ * @author Kris Singh
+ * @author Shikhar Jaiswal
+ *
+ * Tests the RBM Network
+ *
+ * digits dataset source:
```

Comment:
Up to you, but I think we can remove the reference here. I guess it's more likely, that someone will read the class notes or the docs.

Reply:
I'll rather keep it, in case someone wants to find the rationale behind the test.

Reply:
Ah, since we replicate the results from the paper, good idea.